contracts/schemes/Join.sol

```diff
@@ -127,13 +104,12 @@ contract JoinAndQuit is
     returns(bool) {
         Proposal memory proposal = proposals[_proposalId];
         require(proposal.proposedMember != address(0), "not a valid proposal");
-        require(fundings[proposal.proposedMember].state == MemeberState.Candidate, "proposal already been executed");
+        require(membersState[proposal.proposedMember] == MemeberState.Candidate, "proposal already been executed");
```

Comment:
if memberState is "candidate", that means that the proposal is already executed? Maybe change the name of the state to something more meaningful then (like "executed"?)